app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsAdapter.kt

```diff
@@ -46,21 +47,25 @@ class NewsAdapter(private val dataSet: ArrayList<New>) :
         return ViewHolder(view)
     }
 
+    private fun formatTime(date: String): String? {
+        val formatter = DateTimeFormat.forPattern("yyyy-MM-dd")
+        val dateTime = formatter.parseDateTime(date)
+        val prettyTime = PrettyTime(Locale.getDefault())
+        return prettyTime.format(dateTime.toDate())
+    }
+
```

Comment:
Maybe you can make this a String extension function. What do you think?

Reply:
Great idea :P